bin/parse_phenotype.py

```diff
@@ -51,15 +53,17 @@
     desc = {c: template for c in cat} # [Column name: template] looped for column name in category (c = item in list)
     with open(output_dir / f"{name}.json", "w") as f: # Creates file based on f"{value}" ("w", truncates file) saves as 'f'
         json.dump(desc, f, indent=2) # Dumps desc variable into 'f'.json file 
-    cur_df = df[cat].fillna("n/a") # Fills blank cells in selected category of df with 'n/a'
+    cur_df = df[cat].fillna("n/a").set_index("participant_id") # Fills blank cells in selected category of df with 'n/a'
     cur_df.to_csv(output_dir / f"{name}.tsv", sep="\t") # Saves category in .tsv file based on f'{name}' variable
 
 for k in sessions: # Loops the session keys through the following function:
     def sess_parse(sess_key): # Funtion for extracting assessments based on session:
```

Comment:
When defining functions, define it at the top of the script after defining the key informations. that will be at line 48.